finance/proposal-calls/cycle4/orgmycology.md

```diff
@@ -0,0 +1,122 @@
+
+- **Continued Community Manager Support and Coaching** - The OrgMycology team members are available to the community lead on topics including systems automation, community wrangling, qualitative survey techniques and methods and other topics as they arise.
+- **Qualitative Inquiry Coaching** – Supporting Beryl Kanali to continue to grow her skills in interviewing, data gathering and report-out as she has conversations with community members.
+- **Skills Map Development** – Addressing [our recommendation to better communicate user-contributotr pipelines](https://astropy-report.orgmycology.com/sections/consultant_recommendations#develop-and-visualize-user-to-contributor-pipelines). We propose to develop skills maps which can help to map roles of Astropy community members and the ways in which educational material and community progressions take place. We propose to develop two skills maps, one for Astropy Developers (focusing on on-ramps for new developers) and one for Astropy Users (starting with Astropy Learn lessons). These will be developed with community involvement and feedback and shared openly as a resource of the community to own and update as needed. [Examples of skills maps](https://orgmycology.com/skills-maps)
+- **Technical skills support** – Helping and supporting Community Lead as she develops more automations and tools that help her do the work of community manager for the Astropy Community. Project priorities will be set with Community Lead and monthly progress reports will be shared back with the CoCo and community. 
+- **1-2 - Special projects prioritized by CoCo**
+  - Contribution workshops - desing, support and test-out short workshops for PRs, contributing to documentation, learning about automated testing suites etc.
+  - How does Astropy want to work to intentionally welcome more newcomers?
+  - Website strategic facilitations - Drawing from community members vision of a more effective website to be used for a website revamp. 
+
+Across our three person team, we have the domain-specific (open science and OSS) experience and coaching skills to leave the organization in a strong position to get the most out of the community manager role, both in its current ½ time and future full-time capacity. We have extensive experience running interactive online workshops, designing surveys and interview schedules, constructing automations, and building active, participatory communities. We also understand and appreciate that community managers often become overwhelmed by the complex demands of their roles; all of our services are designed to alleviate this pressure by making it easier to gather and act upon feedback from the project community. 
+How does this benefit Astropy packages / users / community / ...? Short summary.
+
+
+Details here as needed for Astropy community members to understand what is planned.
```

Comment:
I agree - details are needed because I don't understand what this will get the astropy community. It seems to be mostly a "support and train the community manager" proposal. Is that the right interpretation?

Reply:
Back in 2023, we had originally proposed that we act as a community manager for a period of time in order to build a foundation for the work of an eventual full-time community manager. This was backed-off of as a suitable candidate was found (and our budget reduced), so we've defaulted to a supporting role in the past year. 

In this proposal, roughly half of this proposal (community manager support, qualitative inquiry coaching, and technical skills support) is continued support for the community manager, and the other half is 2-3  projects that our team will lead, seeking to use to build capability and capacity across Astropy. One of those projects, skills mapping, clearly helps define what community development in both developer and user spaces would look like, this is not a supporting activity per-say though it will support the work of the community manager. 

The other two optional areas, we like to co-create a few activities that are priority areas for the community based on the insights from our survey last year. There is a clear need for Astropy to do more than it is presently doing in areas of concern raised by the community. We're keen to co-create next-steps and tangible actions here. We've observed that this community work is very low on the priority list, and there is a strategic risk for Astropy in continuing to leave it there. With our modest budget, we hope to keep nudging along a few key important areas, and we're keen to co-create what feel most-important to you all at this present moment.